server/rooms.ts

```diff
@@ -248,6 +251,7 @@ export abstract class BasicRoom {
 			title: this.title,
 			auth: Object.create(null),
 			creationTime: Date.now(),
+			section: 'none',
```

Comment:
Why not just default to not having a section entry?

```suggestion
```

Reply:
did